os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -957,20 +1017,29 @@ void up_assert(const uint8_t *filename, int lineno)
 	abort_mode = true;
 #endif
 
+#ifdef CONFIG_FAULT_MGR
+	/* Extract the PC value of instruction which caused the abort/assert */
+
+	if (current_regs) {
+		assert_pc = current_regs[REG_LR];
+	} else {
+		assert_pc = (uint32_t)g_assertpc[0];
+	}
+#endif
+
 #if CONFIG_TASK_NAME_SIZE > 0
 	lldbg("Assertion failed at file:%s line: %d task: %s\n", filename, lineno, this_task()->name);
 #else
 	lldbg("Assertion failed at file:%s line: %d\n", filename, lineno);
 #endif
 	up_dumpstate();
 
-#ifdef CONFIG_BOARD_CRASHDUMP
+#if defined(CONFIG_BOARD_CRASHDUMP) && !defined(CONFIG_FAULT_MGR)
```

Comment:
Is this temporary fix? Do you block dumping?

Reply:
For now I have disable the crash dump feature with fault manager. I am working on to enable this feature with fault manager. Once it is done I will raise a new PR to add the feature with Fault manager.

Reply:
ok